cmd/kubeless/trigger/kafka/create.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright (c) 2016-2017 Bitnami
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kafka
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	kubelessApi "github.com/kubeless/kubeless/pkg/apis/kubeless/v1beta1"
+	"github.com/kubeless/kubeless/pkg/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var createCmd = &cobra.Command{
+
+	Use:   "create <kafka_trigger_name> FLAG",
+	Short: "Create a Kafka trigger",
+	Long:  `Create a Kafka trigger`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		if len(args) != 1 {
+			logrus.Fatal("Need exactly one argument - kafka trigger name")
+		}
+		triggerName := args[0]
+
+		ns, err := cmd.Flags().GetString("namespace")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+		if ns == "" {
+			ns = utils.GetDefaultNamespace()
+		}
+
+		topic, err := cmd.Flags().GetString("trigger-topic")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
+		functionSelector, err := cmd.Flags().GetString("function-selector")
```

Comment:
I would set this as `--function` or `--functions` so people doesn't need to know how we label pods.

Reply:
We are basically passing labels here. Just like kubectl accepts label selector for some of its commands.